src/style.css

```
@@ -1,3 +1,82 @@
 body {
+  background-color: green;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-bg-color: #00ACDC;
+}
+
+html {
+font-family: Roboto, sans-serif;
```

Use 2 spaces for indentation in your fil
```suggestion
  font-family: Roboto, sans-serif;
```